r/src/bpcells-cpp/matrixUtils/Pseudobulk.cpp

```diff
@@ -21,26 +21,29 @@ namespace BPCells {
 template <typename T>
 PseudobulkStats pseudobulk_matrix(std::unique_ptr<MatrixLoader<T>> &&mat,
                                   const std::vector<uint32_t>& cell_groups,
-                                  PseudobulkStatsMethod method,
+                                  int method,
```

Comment:
Not entirely sure if it is 100% worth the syntactic cleanliness, if we have to pass these in now as ints.  I feel like this kind of obfuscates how to use the method argument.